CMakeLists.txt

```diff
@@ -61,6 +61,9 @@ endif()
 option(MTS_ENABLE_PYTHON  "Build Python bindings for Mitsuba, Enoki, and NanoGUI?" ON)
 option(MTS_ENABLE_EMBREE  "Use Embree for ray tracing operations?" OFF)
 option(MTS_ENABLE_GUI     "Build GUI" OFF)
+if (MTS_ENABLE_OPTIX)
+  option(MTS_DYNAMIC_OPTIX  "Resolve CUDA dynamically at run time" ON)
```

Comment:
We always resolve CUDA dynamically at runtime, so this is a bit of a misnomer. I would call it `MTS_USE_OPTIX_HEADERS` "Use OptiX header files instead of resolving GPU ray tracing API ourselves."